### PR TITLE
fix: [BD-38] [TNL-5596][BB-5596] Fix TinyMCE padding

### PR DESCRIPTION
**Pull request by khchine5**
**Supporting information**
- https://openedx.atlassian.net/browse/TNL-5596
- https://tasks.opencraft.com/browse/BB-5596

**Testing instructions**
- Start the DevStack
- Start the Frontend-app-discussions app
- Log in as any user who can create a post/comment and hit `Add a post` button 
- Start writing some content to the text area.
image

### CODE REVIEW

**Comment by khchine5** [comment]
@xitij2000  Fixed.

File: src/components/TinyMCEEditor.jsx [hunk 1]
```diff
@@ -91,6 +91,7 @@ export default function TinyMCEEditor(props) {
           + ' | undo redo',
         content_css: false,
         content_style: contentStyle,
+        body_class: 'm-1',
```
Comment (xitij2000) [inline src/components/TinyMCEEditor.jsx:94]: Let's bump this up to `m-2`

**Review by xitij2000 — APPROVED** [review]
:+1: Just one minor change and this is good to go. 

- [x] I tested this: tested on devstack
- [x] I read through the code

File: src/components/TinyMCEEditor.jsx (hunk 1 above)
Comment (khchine5) [reply src/components/TinyMCEEditor.jsx]: Updated.

**Comment by edx-webhook** [comment]
Hello @khchine5: We are unable to continue with review of your submission at this time. Please see the associated JIRA ticket for more explanation.

**Comment by edx-webhook** [comment]
@khchine5 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-discussions&entry.867055334=https://github.com/openedx/frontend-app-discussions/pull/73&entry.1484655318=https://github.com/khchine5&entry.752974735=2022-02-28+16:41&entry.1917517419=2022-03-18+15:37&entry.2133058324=No) so we can improve your experience in the future.

**Pull request merged**